buildpack.md

```diff
@@ -994,9 +994,9 @@ name = "<dependency name>"
 
 ```toml
 [types]
-  launch = false
-  build = false
-  cache = false
+  launch = true
```

Comment:
I changed all of the flags to true because I feel that it's clearer this way since we omit the table when they're set to false.

Reply:
Please see https://github.com/buildpacks/spec/pull/209/commits/f2eb2af7d05dff04d6570e2057321c45adca96ac.